src/main/java/org/apache/sling/jcr/repoinit/impl/RepositoryInitializerFactory.java

```diff
@@ -177,5 +194,13 @@ protected void applyOperations(Session session, List<Operation> ops, String logM
         }
     }
 
+    
+    protected static boolean isDeveloperModeEnabled() {
```

Comment:
Making this an instance method could help with overriding the flag easier for testing. You can create a sub-class and then override the method to return true not for testing.

Not sure if it's possible with the current set up though, ISTR that the OSGi mocks don't play nicely with subclasses of the actual components.

Reply:
That would leave this method untested. Sure, it's trivial, but testing it does not make the test code much more complicated.
I will leave it as is.